parser.py

```diff
@@ -118,7 +118,20 @@ def parse_mdm_to_nparray(file_name):
 
 # @ Xiang: please implement this function by 09/12
 def dc_iv_input(file_name)
-	_, data = parse_mdm_to_nparray(file_name)
+	header, data = parse_mdm_to_nparray(file_name)
 	# assert whether is it DC IV data
-	# return two numpy array features (voltages) and labels (drain current)
-	return features, labels
\ No newline at end of file
+	# return three numpy arrays, vg, vd and id
+	import numpy as np
+	if ('freq' in data.keys()):
```

Comment:
Writing like this affects the signature of the function. 
Use either assertion or exception. Here you can use:
assert ('freq' not in data.keys()), 'The input data ...'